kibana/templates/configmap.yaml

```diff
@@ -1,12 +1,10 @@
 {{- if .Values.kibanaConfig }}
----
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ template "fullname" . }}-config
+  name: {{ include "kibana.fullname" . }}-config
   labels:
-    app: {{ .Chart.Name }}
-    release: {{ .Release.Name | quote }}
```

Comment:
Removing these labels is a backwards incompatible change. Adding new labels is fine but there is no need to remove these ones.

Reply:
I think these charts are at an early enough stage that we can slip in a few backwards incompatible changes ;) But your call!

Reply:
The reason the charts are following the stack versioning is so that users no they can upgrade without breaking changes. If there is a really good reason to making a breaking change (with a documented way to preserve backwards compatibly) then I would be ok with it. However it would need to have a serious security or availability impact. Removing some labels doesn't really fall into this category. 

I know of a lot of people using labels for filtering and alerting of logs. Someone could upgrade their cluster, things would still work, but suddenly have broken alerting or dashboards because the labels were removed. 

Another even better example is the automated testing done in this repo uses the `release` label. So this change will fail in CI because it can't find which pod to use for testing:

https://github.com/elastic/helm-charts/blob/9dd7bbe2d6bfb113bd63d313063f05c3a024f4dd/helpers/examples.mk#L4